octoprint_domoticz/templates/domoticz_settings.jinja2

```diff
@@ -69,9 +69,19 @@
 			</tr>
 			<tr>
 				<td><div class="controls"><label class="checkbox"><input type="checkbox" data-bind="checked: gcodeEnabled"/> GCODE Trigger</label></div></td>
-				<td><span class="label" title="Use this gcode command to power on relay." data-bind="text: $root.gcodeOnString($data),visible: gcodeEnabled"></span></td>
-				<td><span class="label" title="Use this gcode command to power off relay." data-bind="text: $root.gcodeOffString($data),visible: gcodeEnabled"></span></td>
 			</tr>
+			<tr data-bind="visible: gcodeEnabled">
+				<td colspan="2">
+                    <span class="label" title="Use this gcode command to power on relay." data-bind="text: $root.gcodeOnString($data),visible: gcodeEnabled"></span>
+                    <button type="button" class="btn btn-mini" data-bind="click: function() { copyTextToClipboard($root.gcodeOnString($data)); }">Copy All</button>
```

Comment:
I think I prefer the look of using an icon button for these, similar to the application keys copy button. 

![image](https://github.com/user-attachments/assets/935df805-1c12-4744-89d0-db508b1ed20b)

```
<span class="control-text" title="Use this gcode command to power on relay." data-bind="text: $root.gcodeOnString($data),visible: gcodeEnabled"></span>
<a href="javascript:void(0)" title="Copy GCode to Clipboard" class="control-text" data-bind="click: function() { copyToClipboard($root.gcodeOnString($data)); }"><i class="fas fa-copy"></i></a>
```